www/js/toolpath-displayer.js

```diff
@@ -161,19 +263,19 @@ var transformCanvas = function() {
     // Show the X and Y limit coordinates of the GCode program.
     // We do this before scaling because after we invert the Y coordinate,
     // text would be displayed upside-down.
-    tp.fillStyle = "black";
```

Comment:
I think that the main purpose of these numbers being here is to give a sense of scale so it's obvious if a file is much larger or smaller than intended (ie an inches vs mm mistake). I think that context is better provided by seeing how big the part is on the machine work area.

If we want to bring these back that's OK by me, but there is an issue with them right now where the numbers are only correct in the top down view because of the projection.